lib/Backends/OpenCL/kernels_fwd_conv.cl

```diff
@@ -29,12 +29,25 @@
 // VWM - vector width in dimension M.
 
 static const char* FWD_CONV_CODE = R"(
+#ifndef QUANTIZED
```

Comment:
As I commented above, most of this macro-based magic is not needed anymore, because you have two kernels now.